docs/software/switch-config.md

```diff
@@ -8,6 +8,46 @@ description: To optimize the performance of ExpressLRS, it has different Switch
 !!! warning "WARNING" 
     **Put your arm switch on AUX1**, and set it as **~1000 is disarmed, ~2000 is armed**.
 
+# Summary of Switch Configs
+
+This table summarizes the switch configuration modes and the available switch positions / resolution and update frequency on each channel or flight controller auxiliary channel (Aux X).
+
+| Channel | Flight  <br>Controller | Hybrid | Wide  <br>Hybrid | Full Res  <br>8 | Full Res  <br>16 Half Rate | Full Res  <br>12 Mixed |
+|:---:|:---:|:---:|:---:|:---:|:---:|:---:|
+| 1   | Sticks | **Full Res** | **Full Res** | **Full Res** | Full Res<sup>H</sup> | **Full Res** |
+| 2   | Sticks | **Full Res** | **Full Res** | **Full Res** | Full Res<sup>H</sup> | **Full Res** |
+| 3   | Sticks | **Full Res** | **Full Res** | **Full Res** | Full Res<sup>H</sup> | **Full Res** |
+| 4   | Sticks | **Full Res** | **Full Res** | **Full Res** | Full Res<sup>H</sup> | **Full Res** |
+| **5<sup>+</sup>**   | Aux 1 | **2-pos** | **2-pos** | **2-pos** | **2-pos** | **2-pos** |

```

Comment:
Full Res 16ch has a full res channel 5 (AUX1) as well, it is not 2-pos, but still checked for IsArmed so not very helpful

Reply:
I decided to leave this called out as 2-pos for now so people don't abuse the power of the full resolution arming channel.